tests/filter/squash.t

```diff
@@ -0,0 +1,96 @@
+  $ export RUST_BACKTRACE=1
+  $ git init -q 1> /dev/null
+
+  $ echo contents1 > file1
+  $ git add .
+  $ git commit -m "add file1" 1> /dev/null
+
+  $ git log --graph --pretty=%s
+  * add file1
+
+  $ git checkout -b branch2
+  Switched to a new branch 'branch2'
+
+  $ echo contents2 > file1
+  $ git add .
+  $ git commit -m "mod file1" 1> /dev/null
+
+  $ echo contents3 > file3
+  $ git add .
+  $ git commit -m "mod file3" 1> /dev/null
+
+  $ git checkout master
+  Switched to branch 'master'
+
+  $ echo contents3 > file2
+  $ git add .
+  $ git commit -m "add file2" 1> /dev/null
+
+  $ git merge -q branch2 --no-ff
+
+  $ josh-filter -s --squash "refs/tags/*" --author "New Author" --email "new@e.mail" --update refs/heads/filtered
+  Warning: reference refs/heads/filtered wasn't updated
+  [1] :SQUASH=e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
```

Comment:
weird that it squashes something anyways

Reply:
It does not. This just means that there is an entry in the table.